src/main/java/mate/academy/service/MovieService.java

```diff
@@ -9,4 +9,8 @@ public interface MovieService {
     Movie get(Long id);
 
     List<Movie> getAll();
+
+    Boolean update(Movie movie);
```

Comment:
Consider changing the return type of the `update` method to `void` to better reflect the operation's outcome. Returning a `Boolean` might not be necessary if the method is expected to always succeed or throw an exception otherwise.